docs/en/source/callFlow.rst

```diff
@@ -13,41 +13,33 @@ The Figure is showing the operational flow of entity create/update/append in the
+ 5.1. EM publishes in the Kafka under Topic “ENTITY”.
 
- 5.1. Subscription Manager when getting a notification for the related entity it will check for the notification validation for the current entity and checks if the notification needs to be sent accordingly.
+ 5.2. EM publishes in the Kafka under Topic “Entity_Create/Update/Append” as well.
 
- 5.2. Storage Manager, upon notification from Entities & CR Topics, will trigger the further operations to store/modify the entity related changes in the DB tables. 
+ 5.3. Upon successful pub operation, EM will send the response back.
 
-6. Now entity manager also prepares for registration of the entity data model in the Context Registry. Following are the further functions it performs to achieve the same:
+**Note**: “ENTITY” topic will save all changes of an entity done over a period of time by any of the create/update/append operations of an entity. However, “Entity_Create/Update/Append” Topic (specific to CREATE operation) will only store the data changes of entity create operation only. Having different topics per operation will avoid ambiguity situations among different consumers different requirements. E.g. the subscription manager may need to subscribe for the whole entity, a set of specific attributes, or might be some value change of certain attributes. So, managing all these requirements would be hard if a separate topic per operation is not maintained and would be very simplified to provide direct delta change in data for the given entity at any point in time if separate topics per operation are maintained. Therefore, putting all operations data in a single topic cannot offer the required decoupling, simplification, and flexibility to subscribe/manage at operations, data, or delta data level requirements. So that’s why creating separate topics per operation and one common topic for recording all changes (require to validate the whole entity changes for all operations over a period of time) of all operations to the given entity is the favorable design choice. The context for the given payload is being stored by the LDContext resolver service in the Kafka topic under the name AtContext.
 
- 6.1. So it prepares the csource registration payload (as per NGSI_LD spec section C.3) from the entity payload and fills the necessary field (like id, endpoint as broker IP, location, etc.). Afterword entity manager writes this created csource payload in the CR Topic.
+6.1 Entity topic is published with the same payload so afterwards entity topic writes this created context source payload in the Registry Topic.
 
- 6.2.CR Manager listens to this CR topic and then able to know that some entity has registered.
+6.2 Context Source Registry listens to this registry topic and then able to know that some entity has registered.
 
- 6.3.CR manager writes the updates, if any are there, into the Csource Topic.  
+6.3 Context Source Registry writes the updates, if any are there, into the CSource Topic.  
```

Comment:
[Major] Please changes in flow diagram 1
- Change arrow direction in flow no 6.2
- Add row direction subscription manager to I_REGSUB topic
- Remove CSource topic direction
-  Also changes in flow diagram theoretical part same as figure.

Reply:
Done